bootstrap

```diff
@@ -35,7 +35,7 @@ function check_autotool_prerequisites ()
 {
     # prerequisite versions
     automake_major=1
-    automake_minor=15
+    automake_minor=16
```

Comment:
Nice catch, I'll make a note to do this the next time we change autotools versions.